src/main/java/it/pagopa/pn/deliverypush/action/StartWorkflowHandler.java

```diff
@@ -59,38 +61,59 @@ public StartWorkflowHandler(
      */
     public void startWorkflow(String iun) {
         log.info("Start notification process iun={}", iun);
-        try {
-            NotificationInt notification = notificationService.getNotificationByIun(iun);
+        PnAuditLogBuilder auditLogBuilder = new PnAuditLogBuilder();
+        
+        NotificationInt notification = notificationService.getNotificationByIun(iun);
 
-            try {
-                //Validazione degli allegati della notifica
-                checkAttachmentUtils.validateAttachment(notification);
+        try {
+            //Validazione degli allegati della notifica
+            checkAttachmentUtils.validateAttachment(notification);
 
-                String legalFactId = saveLegalFactsService.saveNotificationReceivedLegalFact(notification);
+            saveNotificationReceivedLegalFacts(notification);
 
-                addTimelineElement(timelineUtils.buildAcceptedRequestTimelineElement(notification, legalFactId), notification);
+            //Start del workflow per ogni recipient della notifica
+            for (NotificationRecipientInt recipient : notification.getRecipients()) {
+                Integer recIndex = notificationUtils.getRecipientIndex(notification, recipient.getTaxId());
+                PnAuditLogEvent logNotificationProcess = auditLogBuilder
+                        .before(PnAuditLogEventType.AUD_NT_INSERT, "Starting notification workflow for recipient={}", recIndex)
```

Comment:
AUD_NT_INSERT è l'inserimento della notifica che è già in pn-delivery quì che cosa di logga?

Reply:
Mi sembra sia un log generico sul processo di workflow di notifica, ha senso? Se non ha senso che ci sia lo tolgo